src/diff/mod.rs

```diff
@@ -247,6 +247,19 @@ impl Diff {
         self.deleted.append(&mut new_files);
         Ok(())
     }
+
+    fn build_path(file: &File, parent_path: &Vec<Label>) -> Path {
+        match parent_path.len() {
+            0 => {
+                // TODO: Path::from_label ?
+                Path::from_labels(file.filename.clone(), &[])
```

Comment:
I believe this would imply that we're at a root folder? So you could do `Path::with_root(&[file.filename.clone()])`

Reply:
I'm afraid this change is in 7fa185957e6ba1d82dd1d37d0c7c3a2fe04f2634 which is not yet merged in master

Reply:
Hmm you may want to `git fetch`, it's there: https://github.com/radicle-dev/radicle-surf/blob/master/src/file_system/mod.rs#L271

Reply:
Yup, missed it, thanks